src/main/resources/application-local.yaml

```diff
@@ -89,7 +93,7 @@ hapi:
     ### the deepest folder level will be used. E.g. - if you put file:/foo/bar/bazz as value then the files are resolved under /static/bazz/**
     #staticLocation: file:/foo/bar/bazz
     ### enable to set the Server URL
-    server_address: https://localhost:8888/oauth/fhir
+    server_address: https://localhost:8080/oauth/fhir
```

Comment:
why are we changing this?

this makes it difficult to bypass auth for local testing since we'll have to go through the proxy

Reply:
AFAIK this is only used to populate the server's address on response payloads. I never had issues testing local going directly to the FHIR server using Postman. What's the scenario that fails to you?

Reply:
Yup exactly, we use the server's address on response payloads when running local, and we don't want to be routed through the proxy by default

Reply:
Everything is paginated by default now, so this address is always used

Reply:
Taken to Slack: https://metriport.slack.com/archives/C04CXLSAF7V/p1683741124250499?thread_ts=1683701173.918069&cid=C04CXLSAF7V